lib/client.js

```diff
@@ -0,0 +1,12 @@
+const { Client } = require('@elastic/elasticsearch');
+const { URL } = require('url');
+
+module.exports = new Client({
+  node: {
+    url: new URL('http://localhost:9200'),
+    auth: {
+      username: 'elastic',
+      password: 'changeme',
+    },
+  },
+});
```

Comment:
Everything is hardcoded here, is it normal ?

Reply:
yep, but this file should be in the test part